### PR TITLE
PR: Disambiguate tabs

### LINKED ISSUE #39: Disambiguate report tabs
As we do in the Editor and the IPython console, we should also disambiguate report tabs if two reports have the same name.

Please take a look at the `disambiguate_fname` method in `plugins/ipythonconsole.py` to see how to do it.

### PR DESCRIPTION
**Pull request by rlaverde**
Fixes #39

### CODE REVIEW

File: spyder_reports/widgets/tests/test_report_widget.py [hunk 1]
```diff
@@ -37,6 +38,21 @@ def setup_reports(qtbot):
     return widget
 
 
+@pytest.fixture
+def setup_reports_close_tab(qtbot):
```
Comment (ccordoba12) [inline spyder_reports/widgets/tests/test_report_widget.py]: Please add a docstring here

File: spyder_reports/reportsplugin.py [hunk 2]
```diff
@@ -36,7 +36,7 @@ class ReportsPlugin(SpyderPluginWidget):
 
     CONF_SECTION = 'reports'
     CONFIGWIDGET_CLASS = None
-    sig_render_finished = Signal()
+    sig_render_finished = Signal(bool, object, object)
```
Comment (ccordoba12) [inline spyder_reports/reportsplugin.py]: Why did you change the signature of this signal?

File: spyder_reports/widgets/tests/test_report_widget.py (hunk 1 above)
Comment (ccordoba12) [inline spyder_reports/widgets/tests/test_report_widget.py]: That's why CircleCI is failing ;-)

File: spyder_reports/reportsplugin.py (hunk 2 above)
Comment (rlaverde) [inline spyder_reports/reportsplugin.py]: This signal returns the if the render was success, the filename, and the errors, I added this parameters for using in tests

Comment (goanpeca) [inline spyder_reports/reportsplugin.py]: @rlaverde you should add a comment to clearly explain what each `object` is
